SPECS-EXTENDED/podman/podman.spec

```diff
@@ -447,262 +217,181 @@ that dnsmasq will read in.  Similarly, when a pod
 
 mv pkg/hooks/README.md pkg/hooks/README-hooks.md
 
-# install plugins
-pushd dnsname-%{commit_plugins}
+# install dnsname plugin
+cd %{repo_plugins}-%{commit_plugins}
 %{__make} PREFIX=%{_prefix} DESTDIR=%{buildroot} install
-popd
+cd ..
+
+# install gvproxy
+cd %{repo_gvproxy}-%{commit_gvproxy}
+install -dp %{buildroot}%{_libexecdir}/%{name}
+install -p -m0755 bin/gvproxy %{buildroot}%{_libexecdir}/%{name}
+cd ..
 
 # do not include docker and podman-remote man pages in main package
 for file in `find %{buildroot}%{_mandir}/man[15] -type f | sed "s,%{buildroot},," | grep -v -e remote -e docker`; do
     echo "$file*" >> podman.file-list
 done
 
-# do not install remote manpages on centos7
-%if ! 0%{?fedora} && 0%{?centos} < 8
-rm -f %{buildroot}%{_mandir}/man1/docker-remote.1
-rm -f %{buildroot}%{_mandir}/man1/%{name}-remote.1
-rm -f %{buildroot}%{_mandir}/man5/%{name}-remote.conf.5
-rm -f %{buildroot}%{_datadir}/bash-completion/completions/%{name}-remote
-rm -f %{buildroot}%{_datadir}/zsh/site-functions/_%{name}-remote
-rm -f %{buildroot}%{_datadir}/fish/vendor_completions.d/%{name}-remote.fish
-rm -f %{buildroot}%{_tmpfilesdir}/%{name}.conf
-%endif
-
-# source codes for building projects
-%if 0%{?with_devel}
-install -d -p %{buildroot}/%{gopath}/src/%{import_path}/
-
-echo "%%dir %%{gopath}/src/%%{import_path}/." >> devel.file-list
-# find all *.go but no *_test.go files and generate devel.file-list
-for file in $(find . \( -iname "*.go" -or -iname "*.s" \) \! -iname "*_test.go" | grep -v "vendor") ; do
-    dirprefix=$(dirname $file)
-    install -d -p %{buildroot}/%{gopath}/src/%{import_path}/$dirprefix
-    cp -pav $file %{buildroot}/%{gopath}/src/%{import_path}/$file
-    echo "%%{gopath}/src/%%{import_path}/$file" >> devel.file-list
-
-    while [ "$dirprefix" != "." ]; do
-        echo "%%dir %%{gopath}/src/%%{import_path}/$dirprefix" >> devel.file-list
-        dirprefix=$(dirname $dirprefix)
-    done
-done
-%endif
-
-# testing files for this project
-%if 0%{?with_unit_test} && 0%{?with_devel}
-install -d -p %{buildroot}/%{gopath}/src/%{import_path}/
-# find all *_test.go files and generate unit-test-devel.file-list
-for file in $(find . -iname "*_test.go" | grep -v "vendor") ; do
-    dirprefix=$(dirname $file)
-    install -d -p %{buildroot}/%{gopath}/src/%{import_path}/$dirprefix
-    cp -pav $file %{buildroot}/%{gopath}/src/%{import_path}/$file
-    echo "%%{gopath}/src/%%{import_path}/$file" >> unit-test-devel.file-list
-
-    while [ "$dirprefix" != "." ]; do
-        echo "%%dir %%{gopath}/src/%%{import_path}/$dirprefix" >> devel.file-list
-        dirprefix=$(dirname $dirprefix)
-    done
-done
-%endif
-
-%if 0%{?with_devel}
-sort -u -o devel.file-list devel.file-list
-%endif
+rm -f %{buildroot}%{_mandir}/man5/docker*.5
 
 install -d -p %{buildroot}/%{_datadir}/%{name}/test/system
 cp -pav test/system %{buildroot}/%{_datadir}/%{name}/test/
 
-%check
-%if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
```

Comment:
Please bring back the contents for the `%check` sections. Even if they are disabled right now, they will be a good starting point for enabling them in the future.

Similar request for `podman.spec`.

Reply:
I didn't remove. The change was made as per latest version podman.